### DISCUSSION TITLE
Theme: make theme selection a user scope configuration.

### DISCUSSION THREAD

**Discussion by kevisong** [grafana/grafana#41701] [General]
The light and dark UI theme selection is now at organization level.

It would be better if this can be a user level configuration that different user can have either light or dark theme.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
